.github/workflows/main.yml

```diff
@@ -0,0 +1,36 @@
+name: build
+on: [pull_request, push]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        # Use these Java versions
+        java: [
+          1.8,  # Minimum supported by Minecraft
+          11,   # Current Java LTS
+          15    # Latest version
+        ]
+        # and run on both Linux and Windows
+        os: [ubuntu-20.04, windows-latest]
```

Comment:
running on windows has no effect and iirc github charges money for it.

Reply:
spottedleaf has pro, which gets you 2000 minutes of free actions

Reply:
![image](https://user-images.githubusercontent.com/43724816/107194959-fd706080-69f0-11eb-82eb-53a39d212e8f.png)
😅

Reply:
building on windows is still unnecessary

Reply:
I agree  with not OM3GA, there is no reason to compile on windows and linux when dealing with java, its cross-platform